### ISSUE TITLE
WCAG compliance overview

### ISSUE THREAD

**Issue by thiessenp-cds** [cds-snc/platform-forms-client#3735]
Do a WCAG AA test on the conditional logic screens. Mark down any success criteria failures.

Note: the manual testing part with screen readers etc. will be done in another ticket.

WCAG Error List:
- [x]  More button missing label. Has “more” tooltip but needs a label. Also, low contrast
- [x]  Tab labels have nested IMG with alt-text. (false positive I think?)
- [x]  Right panel Inactive Tab labels fail minimum contrast
- [x]  Questions Screen, question input missing a label/aria-label..
- [x]  Logic Screen, “?” question button needs an svg-title/alt-text
- [x]  Logic Screen, an empty label (see image - hard to see)
- [x]  Tree view, aria-attributes could be improved (maybe later, would require forking lib)
- [x]  Tree view, tree children ul/li elements need to match role (maybe later, would require forking lib)
- [x]  Logic language toggle, aria-attributes could be improved (Best alternative I think)
- [x]  Add live-regions on right panel tree-view tab panes

**Comment by thiessenp-cds** [comment]
A first pass resulted in these errors.
- [ ] More button missing label. Has “more” tooltip but needs a label. Also, low contrast
- [ ] Tab labels have nested IMG with alt-text. (false positive I think?)
- [ ] Right panel Inactive Tab labels fail minimum contrast
- [ ] Questions Screen, question input missing a label/aria-label.. 
- [ ] Logic Screen,  “?” question button needs an svg-title/alt-text
- [ ] Logic Screen, an empty label (see image - hard to see)
- [ ] Tree view, aria-attributes could be improved
- [ ] Tree view, tree children ul/li elements need to match role 
- [ ] Logic language toggle, aria-attributes could be improved

**Comment by thiessenp-cds** [comment]
I tested the forms-form page also. No WCAG issues found.
